python/cuml/decomposition/tsvd_mg.pyx

```diff
@@ -253,7 +253,7 @@ class TruncatedSVDSPMG:
             raise TypeError(msg)
 
         if (not np.isfortran(X)):
-            X = np.array(X, order='F')
+            X = np.array(X, order='F', dtype=X.dtype)
```

Comment:
This just views the data as column major and doesn't actually transform it, correct? Is that the expected behavior?

Reply:
Nevermind, this actually copies, transforms, and typecasts the data 👍

Reply:
Just as a note, could also use `np.require(X, requirements='F')`, which has the same behavior, but avoids the need to add `dtype`.

ref: https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.require.html